src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -398,6 +398,7 @@ private function generateProxyClasses()
             if (!$proxyDumper->isProxyCandidate($definition)) {
                 continue;
             }
+            $this->container->getReflectionResource($definition->getClass());
```

Comment:
Where does this method come from and what is it going? 🤔

Reply:
From the moon of course! comment added :)

Reply:
That's weird, I can't find its definition anywhere. Maybe only on GitHub's moon version? 😄 

Though, registering something by calling a getter is weirdo.

Reply:
oups, bad method name :) fixed

Reply:
Aaaah, makes much more sense!